src/components/textbox/textbox.pw.tsx

```diff
@@ -0,0 +1,1244 @@
+
+    await expect(textbox(page)).toHaveCSS("flex-direction", "row");
+  });
+
+  /* This test could be deleted, as its specific test case cannot be properly replicated using Playwright,
+  we can assert it exists in the DOM in the specified DOM structure, but cannot properly test its visibility
+  without added complexity or adding new data tags to `Textbox`s DOM. */
+
+  ([
+    [true, "input", "label"],
+    [false, "label", "input"],
+  ] as [boolean, string, string][]).forEach(
+    ([boolean, firstElement, secondElement]) => {
+      test(`should render with reverse prop set to ${boolean}`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<TextboxComponent reverse={boolean} />);
+
+        const textboxParentParent = await textbox(page).evaluateHandle(
+          (element: Element) => {
+            return element.parentElement?.parentElement;
+          }
+        );
+
+        const domElement1 = await page.evaluate(
+          ({
+            textboxParentParent: parentDomElement,
+            firstElement: element,
+          }) => {
+            const returnedElement = parentDomElement?.querySelectorAll(element)
+              .length;
+            return returnedElement;
+          },
+          { textboxParentParent, firstElement }
+        );
+
+        const domElement2 = await page.evaluate(
+          ({
+            textboxParentParent: parentDomElement,
+            secondElement: element,
+          }) => {
+            const returnedElement = parentDomElement?.querySelectorAll(element)
+              .length;
+            return returnedElement;
+          },
+          { textboxParentParent, secondElement }
+        );
+
+        expect(domElement1).toBeTruthy();
+        expect(domElement2).toBeTruthy();
+      });
+    }
+  );
+
+  /* This test could be deleted, as its specific test case cannot be properly replicated using Playwright,
+  we can assert it exists in the DOM in the specified DOM structure, but cannot properly test its visibility
+  without added complexity or adding new data tags to `Textbox`s DOM. */
+
+  test("should render with positionedChildren prop", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<TextboxComponentWithPositionedChildren />);
+
+    const textboxParent = await textbox(page).evaluateHandle(
+      (element: Element) => {
+        return element.parentElement;
+      }
+    );
+
+    const buttonExistsInTextboxParent = await page.evaluate(
+      (parentDomElement) => {
+        const element = parentDomElement?.querySelectorAll(
+          `[data-component="button"]`
+        ).length;
+        return element;
+      },
+      textboxParent
+    );
+
+    expect(buttonExistsInTextboxParent).toBeTruthy();
+  });
+
+  ([
+    ["flex", 399],
+    ["flex", 400],
+    ["block", 401],
+  ] as [string, TextboxProps["adaptiveLabelBreakpoint"]][]).forEach(
+    ([displayValues, breakpoint]) => {
```

Comment:
```suggestion
    ([displayValue, breakpoint]) => {
```

Again, let's use singular names for single values.